homeassistant/components/broadlink/remote.py

```diff
@@ -64,6 +69,9 @@
 SERVICE_LEARN_SCHEMA = COMMAND_SCHEMA.extend(
     {
         vol.Required(ATTR_DEVICE): vol.All(cv.string, vol.Length(min=1)),
+        vol.Optional(ATTR_COMMAND_TYPE, default=COMMAND_TYPES[0]): vol.In(
```

Comment:
Use the constant `COMMAND_TYPE_IR` directly instead of using the list of command types.
```suggestion
        vol.Optional(ATTR_COMMAND_TYPE, default=COMMAND_TYPE_IR): vol.In(
```